src/custom/pages/Faq/index.tsx

```diff
@@ -573,6 +556,102 @@ export default function Faq() {
+            and investing into ecosystem projects.
+          </p>
+          <p>
+            Staking - CoW Protocol solvers will be incentivized to compete for providing best solutions by capturing
+            value for participation. This could come in the form of trading fees, solution token rewards or both.
+            Solvers could be required to stake COW in order to participate in solution submission. CoW Protocol users
+            will be able to lock COW tokens for a fee discount on their trades. This gives tangible utility to the
+            tokens benefiting all CoW trades.
+          </p>
+          <p>
+            Fees and Rewards - Traders, solvers, frontends, developers and other ecosystem participants should always
+            remain sufficiently incentivized to continue contributing and maintaining a robust ecosystem. Incentives
+            could come in the form of batch solution rewards, trader incentives, ecosystem grants and more.
+          </p>
+          <h3 id="what-was-the-criteria-for-the-token-airdrop">What was the criteria for the token airdrop?</h3>
+          <p>There are 4 categories in which you can qualify for the airdrop:</p>
+          <p>
+            CoW POAP: Awarded POAPs from the testing sessions, Batch challenges, CowStars or user research sessions. Not
+            all are equally valuable. Some that are most special are the ones for our CowStars and User Test
+            Participants. Note that we say “awarded” and not “owns”, since we know that there have been secondary
+            markets for these POAPS. We only took awardees into consideration.
+          </p>
+          <p>
+            Power Traders: A Power trader is someone who has traded on either Ethereum mainnet (before block 13974427)
+            or Gnosis Chain (block 20024195) a minimum of 3 trades, at least 14 days between the 1st and the last trade,
+            and for a minimum of 1K total volume. Note that stable to stable coin trades only by factor 0.1
+          </p>
+          <p>
+            Traders: A trader is someone who has traded on either Ethereum mainnet (before block 13974427) or Gnosis
+            Chain (block 20024195) at least once for a total of 1K USD or who has done at least 5 trades. Note that
+            stable to stable coin trades only by factor 0.1.
+          </p>
+          <p>
+            GNO Holders: A GNO holder is someone who held at least 0.1 GNO on either Ethereum mainnet (before block
+            13974427) or Gnosis Chain (block 20024195). They were eligible for holding GNO on Mainnet and/or
+            GnosisChain, and for running GBC validators. In addition, all the following LP token holders were accounted
+            for: Balancer v2 (mainnet), Uniswap v3 (mainnet), Honeyswap (gnosischain), Symetric (gnosischain), Sushiswap
+            (gnosischain), Elk (gnosischain), Swapr (Gnosischain).
```

Comment:
Maybe we should use lower case on the last `(Gnosischain).` for consistency :smile: